### PR TITLE
IGNITE-11874 Fix mismatch between idle_verify results with and without -dump option.

### PR DESCRIPTION
**Pull request by denis-chudov**

### CODE REVIEW

File: modules/core/src/test/java/org/apache/ignite/util/GridCommandHandlerAbstractTest.java [hunk 1]
```diff
@@ -200,4 +209,48 @@ protected void addExtraArguments(List<String> args) {
...
+                        && ((GridNearTxPrepareFutureAdapter) fut).tx().system())
+                        continue;
+
+                    if (fut instanceof GridDhtTxPrepareFuture
+                        && ((GridDhtTxPrepareFuture) fut).tx().system())
+                        continue;
+
+                    log.error("Expecting no active future [node=" + ig.localNode().id() + ", fut=" + fut + ']');
+
+                    hasFutures = true;
```
Comment (vmalin) [inline modules/core/src/test/java/org/apache/ignite/util/GridCommandHandlerAbstractTest.java]: It is possible to move {fail} here and remove hasFutures varaible

File: modules/core/src/test/java/org/apache/ignite/util/GridCommandHandlerAbstractTest.java [hunk 2]
```diff
@@ -200,4 +209,48 @@ protected void addExtraArguments(List<String> args) {
...
     }
+
+    /**
+     * Checks if all non-system txs and non-system mvcc futures are finished.
+     */
+    protected void checkUserFutures() {
+        for (Ignite ignite : G.allGrids()) {
+            IgniteEx ig = (IgniteEx)ignite;
+
+            final Collection<GridCacheFuture<?>> futs = ig.context().cache().context().mvcc().activeFutures();
```
Comment (vmalin) [inline modules/core/src/test/java/org/apache/ignite/util/GridCommandHandlerAbstractTest.java]: Please add more descriptive variable name. mvccFuts of something like this

Comment (denis-chudov) [reply modules/core/src/test/java/org/apache/ignite/util/GridCommandHandlerAbstractTest.java]: fixed

**Comment by dspavlov** [comment]
IGNITE-11874 | Resolved
-- | --

**Pull request closed without merging**